code/modules/research/xenoarchaeology/artifact/artifact_necromarker.dm

```diff
@@ -0,0 +1,86 @@
+#define NECROMARKER_MIN_WHISPER_INTERVAL 200
+#define NECROMARKER_MAX_WHISPER_INTERVAL 450
+/obj/machinery/necromarker
+    name = "mysterious sculpture"
+    desc = "A mysterious scultpure of spiral spines, covered in runes."
+    icon = 'icons/xenoarch_icons/necromarker.dmi'
+    icon_state = "black"
+    density = 1
+
+    var/ticks_not_whispered = 0
+    var/next_whisper = 300
+    var/whispers = list("...bring me flesh...", "...make us whole...", "...we must be whole...", "...join us in unity...", "...one mind, one soul, one flesh...", "...MAKE US WHOLE...")
```

Comment:
shouldn't this be "bring us flesh" to keep in the same tune as the rest of the whispers?